.tekton/assisted-service-mcp-saas-main-push.yaml

```
@@ -530,6 +534,9 @@ spec:
         value: $(tasks.build-image-index.results.IMAGE_URL)
       - name: IMAGE_DIGEST
         value: $(tasks.build-image-index.results.IMAGE_DIGEST)
+      - name: ADDITIONAL_TAGS
+        value: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)